tasks/destroy-volumes.yml

```diff
@@ -3,8 +3,9 @@
   script: >
     destroy_virt_volume.sh
     {{ item.name }}
-    {{ item.pool }}
+    {{ item.pool | default('default') }}
   with_items: "{{ volumes }}"
+  when: item.type | default(libvirt_volume_default_type) in ['volume', 'network']
```

Comment:
We only run `virt_volume.sh` when `item.type | default(libvirt_volume_default_type) == 'volume'` (https://github.com/stackhpc/ansible-role-libvirt-vm/blob/9fab5e65ec46e04c2b010338015592e0713a3f3a/tasks/volumes.yml#L32). Just wondering if destroy should match that. As far as I can determine, the network volumes aren't defined in libvirt as a volume, but are just referenced in the VM definition.

Reply:
Must have been asleep this morning, I could swear I checked that.